### ISSUE TITLE
Remove FFaker gem

### ISSUE THREAD

**Issue by MinasMazar** [solidusio-contrib/solidus_extension_dev_tools#24]
Since [Solidus 2.7 does not need ffaker gem](https://github.com/solidusio/solidus_support/pull/15#issuecomment-446900986) anymore
and Solidus 2.6 will not be supported from 2019.12.16 (see [here](https://solidus.io/security/)), we should remove it.

**Comment by aldesantis** [comment]
Agreed, we should remove FFaker once we update the orb to build with Solidus 2.7+.
